src/disc/stk/Albany_STKDiscretization.cpp

```diff
@@ -457,7 +457,7 @@ STKDiscretization(const Teuchos::RCP<Teuchos::ParameterList>&  discParams_,
  , discParams(discParams_)
  , interleavedOrdering(stkMeshStruct_->interleavedOrdering)
 {
-  // Nothing to be done here
+  this->updateMesh(); 
```

Comment:
This is wrong. The method `updateMesh` calls virtual methods, which C++ does not allow (or, better, it allows it, but it won't do what you expected). This call got into the constructor somehow a while ago, I took it out again in a recent commit.